Modules/Numerics/Eigen/wrapping/itkEigenAnalysis2DImageFilter.wrap

```diff
@@ -1,5 +1,11 @@
 itk_wrap_class("itk::EigenAnalysis2DImageFilter" POINTER)
+  # Only vectors with 2 components are valid. Temporarily change the global vector dims list.
```

Comment:
Of the few update class wrap files, this filter seems to be the only one which is 2D-only. So these changes make sense.